web-api/src/public-api/casePublicSearchLambda.js

```diff
@@ -15,8 +15,5 @@ exports.casePublicSearchLambda = event =>
         ...event.queryStringParameters,
       });
     },
-    {
-      isPublicUser: true,
-      user: {},
-    },
+    { user: {} },
```

Comment:
`isPublicUser` was used for logging purposes only. Now, the logger checks if the user is `{}`, which is the same check other parts of the application use to determine if it is a public user.